mobile_codes/json/countries.json

```diff
@@ -2,7 +2,7 @@
     ["Afghanistan", "AF", "AFG", "004", "412"],
     ["Albania", "AL", "ALB", "008", "276"],
     ["Algeria", "DZ", "DZA", "012", "603"],
-    ["American Samoa", "AS", "ASM", "016", "544"],
+    ["American Samoa", "AS", "ASM", "016", ["311", "544"]],
```

Comment:
How to read this:

* Country name: ``American Samoa``
* [ISO 3166-1 alpha-2 code](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2): ``AS``
* [ISO 3166-1 alpha-3 code](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-3): ``ASM``
* [ISO 3166-1 numeric code](https://en.wikipedia.org/wiki/ISO_3166-1_numeric): ``016``
* [Mobile Country Codes](https://en.wikipedia.org/wiki/Mobile_country_code): 
  - ``311``: Generally United states, but used by American Samoa Telecommunications
  - ``544``: National operator, Bluesky in American Samoa